nilearn/decomposition/tests/test_canica.py

```diff
@@ -1,8 +1,11 @@
 """Test CanICA"""
-import nibabel
+
 import numpy as np
 from numpy.testing import assert_array_almost_equal
-from nose.tools import assert_true
+from nose.tools import assert_true, assert_raises
+
+import nibabel
```

Comment:
I tend to try and reorganize imports when I add them (3 blocks, alphabetized, `import`s before `from`.

I need to stop doing that, it's not helping anybody.  I'll revert this now.